app/views/content/train-to-be-a-teacher/who-do-you-want-to-teach.md

```diff
@@ -22,6 +22,8 @@ England.
 
 You will need a degree to teach all student groups except those in further education, depending on the type of further education you are teaching.
 
+Find out about the [support you can get while training to teach if you're disabled](/funding-and-support/if-youre-disabled). 
```

Comment:
I think this is _slightly_ awkward here - it doesn't quite fit with the rest of the content in this section?

Would it be better as a related content link, as on `/train-to-be-a-teacher/if-you-have-a-degree` ?

Happy for you both to disagree though!